From 718eaabf024e0108c9443e0864dfb500e58d61cf Mon Sep 17 00:00:00 2001
From: Felix Radensky <felix.r@variscite.com>
Date: Thu, 3 Dec 2020 18:10:24 +0200
Subject: [PATCH 25/31] usb: gadget: Fix ethernet gadget compilation failure

When CONFIG_USB_ETHER is defined but CONFIG_SPL_USB_ETHER is not
the SPL complation fails. This patch fixes the problem.

Signed-off-by: Nate Drude <nate.d@variscite.com>
---
 drivers/usb/gadget/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/Makefile b/drivers/usb/gadget/Makefile
index 70f3bf43e7..40a0cc6394 100644
--- a/drivers/usb/gadget/Makefile
+++ b/drivers/usb/gadget/Makefile
@@ -31,7 +31,7 @@ obj-$(CONFIG_USB_FUNCTION_SDP) += f_sdp.o
 obj-$(CONFIG_USB_FUNCTION_ROCKUSB) += f_rockusb.o
 endif
 endif
-ifdef CONFIG_USB_ETHER
+ifdef CONFIG_$(SPL_)USB_ETHER
 obj-y += ether.o
 obj-$(CONFIG_USB_ETH_RNDIS) += rndis.o
 obj-$(CONFIG_CI_UDC)	+= ci_udc.o
-- 
2.25.1

